lib/active_triples/property_builder.rb

```diff
@@ -9,7 +9,10 @@ def initialize(name, options, &block)
     end
 
     def self.create_builder(name, options, &block)
-      raise ArgumentError, "property names must be a Symbol" unless name.kind_of?(Symbol)
+      raise ArgumentError, "property names must be a Symbol" unless
+        name.kind_of?(Symbol)
+      raise ArgumentError, "property names must be a Symbol" unless
```

Comment:
Good catch. Copy paste is fun. Will fix.
On Feb 19, 2015 4:30 PM, "Trey Terrell" <notifications@github.com> wrote:

> In lib/active_triples/property_builder.rb
> <https://github.com/ActiveTriples/ActiveTriples/pull/103#discussion_r25040617>
> :
>
> > @@ -9,7 +9,10 @@ def initialize(name, options, &block)
> >      end
> >
> >      def self.create_builder(name, options, &block)
> > -      raise ArgumentError, "property names must be a Symbol" unless name.kind_of?(Symbol)
> > +      raise ArgumentError, "property names must be a Symbol" unless
> > +        name.kind_of?(Symbol)
> > +      raise ArgumentError, "property names must be a Symbol" unless
>
> That doesn't seem like the right error message.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ActiveTriples/ActiveTriples/pull/103/files#r25040617>.
>